test/optimization/test_min_eigen_optimizer.py

```diff
@@ -79,12 +79,8 @@ def test_min_eigen_optimizer(self, config):
 
             # analyze results
             self.assertAlmostEqual(cplex_result.fval, result.fval)
-        except RuntimeError as ex:
-            msg = str(ex)
-            if 'CPLEX' in msg:
-                self.skipTest(msg)
-            else:
-                self.fail(msg)
+        except NameError as ex:
```

Comment:
This might be a bit pedantic, but maybe we should check if the name error is actually coming from a missing  CPLEX. I already ran into a case with the `GroverOptimizer` where the tests were wrongly skipped.

Reply:
Updated except test.